lib/rules/no-restricted-v-on.js

```diff
@@ -0,0 +1,192 @@
+/**
+ * @author Kamogelo Moalusi <github.com/thesheppard>
+ * See LICENSE file in root directory for full license.
+ */
+'use strict'
+
+const utils = require('../utils')
+const regexp = require('../utils/regexp')
+
+const DEFAULT_OPTIONS = [
+  {
+    argument: '/^v-/',
+    message:
+      'Using `@v-xxx` is not allowed. Instead, remove `@` and use it as directive.'
```

Comment:
I don't think this is a common use case, and certainly not a valid suggestion. Please remove the default option altogether (also from the docs). So by default, the rule should do nothing.